src/make.js

```diff
@@ -0,0 +1,13 @@
+module.exports.make = (function f() {
+  let params = [];
+  return function calc(...args) {
+    if (typeof args[0] === 'function') {
+      if (params.length === 0) return undefined;
```

Comment:
**params.length === 0** the same as **!params.length**
**return undefined** the same as **return**

It is a good practice to always use block statements after `if` statement - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else#Description